src/locales/pt-BR/admin.ftl

```diff
@@ -222,6 +223,8 @@ configure-auth-sso-key = Chave
 configure-auth-sso-regenerate = Regerar
 configure-auth-sso-regenerateAt = CHAVE GERADA EM:
   { DATETIME($date, year: "numeric", month: "numeric", day: "numeric", hour: "numeric", minute: "numeric") }
+  configure-auth-sso-regenerateHonoredWarning =
```

Comment:
Whitespace is unfortunately an issue in FTL files. Ensure translation keys have no leading whitespace.

```suggestion
configure-auth-sso-regenerateHonoredWarning =
```